src/mlpack/tests/knn_test.cpp

```diff
@@ -1405,13 +1405,17 @@ BOOST_AUTO_TEST_CASE(MoveConstructorRTreeTest)
 
   knn2.Search(3, neighbors2, distances2);
 
+  // Use move assignment.
+  NeighborSearchType knn3 = std::move(knn2);
+  knn3.Search(3, neighbors3, distances3);
+
   CheckMatrices(neighbors, neighbors2);
   CheckMatrices(distances, distances2);
```

Comment:
Looks like you have added the code but you failed to check the matrices.

Reply:
Oh wow, can't believe I missed that. Will add it in :)